docs/resources/client/level.md

```diff
@@ -6,37 +6,39 @@ A level is a playable object in Geometry Dash, namely coming with data that expl
-| k20 | Practice Mode Percentage | **integer**                                                      | the max percentage that has been achieved in practice mode in this level         |
-| k22 | Like Rating              | **integer**                                                      | the level's like rating (`likes - dislikes`)                                     |
-| k23 | Length                   | **[Length](enumerations.md)**                                    | the level's length                                                               |
-| k26 | Stars                    | **integer**                                                      | the stars the level is worth                                                     |
-| k41 | Password                 | **integer**                                                      | the [password]() <!-- local gamesave password topic link --> set for the level   |
-| k45 | Custom Song ID           | **integer**                                                      | the custom Song ID (if used)                                                     |
-| k46 | Level Revision           | **integer**                                                      | the revision of the level                                                        |
-| k48 | Object Count             | **integer**                                                      | the [object count]() of the level                                                |
-| k61 | First Coin Acquired      | **bool**                                                         | whether the first coin is acquired during verification                           |
-| k62 | Second Coin Acquired     | **bool**                                                         | whether the second coin is acquired during verification                          |
-| k63 | Third Coin Acquired      | **bool**                                                         | whether the third coin is acquired during verification                           |
-| k66 | Requested Stars          | **integer**                                                      | the requested stars during publication of the level                              |
-| k71 | Orb Reward Percentage    | **integer**                                                      | the percentage up until the orb reward has been granted                          |
-| k79 | Unlisted                 | **bool**                                                         | whether the level is to be marked as unlisted or not during publication          |
-| k80 | Seconds Spent Editing    | **integer**                                                      | the number of seconds spent editing the level                                    |
-| k83 | Difficulty               | **[Difficulty](enumerations.md)**                                | the level difficulty (including Demon ratings)                                   |
```

Comment:
If it's a wrong register, however discovered in gamesaves, consider including it in the current unknown values section below.